scripts/host.sh

```diff
@@ -0,0 +1,2 @@
+(cd ../judging-backend ; node &> /dev/null) &
```

Comment:
I could be wrong, but doesn't `node` just start the node interpreter? This should be `node .`, no?

Reply:
`node` should automatically run index.js in the current directory, but we can change it to `node .` just for clarity.